unit-tests/post-processing/test-post-processing-from-bag.py

```diff
@@ -0,0 +1,201 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+#temporary fix to prevent the test from running on Win_SH_Py_DDS_CI 
+#test:donotrun:dds
+
+import pyrealsense2 as rs
+from rspy import test
+import numpy as np
+import tempfile, os.path
+import threading
+import time
+################################################################################################
+
+frame_processor_lock = threading.Lock()
+frames_lock = threading.Lock()
+completion_event = threading.Event()
+all_paris = True
+composite_frames = []
+sensor_to_frames = {}
+sensor_to_framesets = {}
+sensor_to_frame = {}
+
+
+class AlignRecordBlock:
+
+    def __init__(self, align_to, align_from):
+        self._align_from = rs.stream(align_from)
+        self._align = rs.align(align_to)
+
+    def record(self, sensor_name, frame, sctx):
+        ss = sctx.sw_sensors[sensor_name]
+        ss.on_video_frame(create_sw_frame(frame, sctx.sw_stream_profiles[sensor_name][self._align_from]))
+        ss.stop()
+        ss.close()
+
+    def process(self, frame):
+        fs = frame.as_frameset()
+        return self._align.process(fs).first_or_default(self._align_from)
+
+
+def create_sw_frame(video_frame, profile):
+    new_video_frame = rs.software_video_frame()
+    new_video_frame.data = np.array(np.hstack(np.asarray(video_frame.get_data(), dtype=np.uint8)), dtype=np.uint8)
```

Comment:
This is not readable.
And it's using numpy, which I'd like to avoid.
What does it do? Can't we just use `bytearray`?